js/activity.js

```diff
@@ -1889,6 +1885,10 @@ define(function (require) {
                 errorArtwork['zerodivide'].visible = true;
                 stage.setChildIndex(errorArtwork['zerodivide'], stage.getNumChildren() - 1);
                 break;
+  	   case NOINPUTERRORMSG:
```

Comment:
....It's not fixed. My editor & github has indentation inconsistensies somehow. Let me give it another look.